test/kitchen/test/integration/dd-security-agent-test/rspec/dd-security-agent-test_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe 'successfully run functional test' do
+  it 'displays PASS and returns 0' do
+    output = `sudo /tmp/security-agent/testsuite -test.v 1>&2`
+    retval = $?
+    expect(retval).to eq(0)
```

Comment:
Do you want to also add `expect(output).not_to include("FAIL")`  like in the following `describe` ?

Reply:
I don't think this is useful. If one test fails, it will return something different from 0